gallery/src/styles/NxVulnerabilityIndicator/NxVulnerabilityIndicatorPage.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+const nxVulnerabilityIndicatorCode = require('./NxVulnerabilityIndicatorExample.html');
+
+const NxVulnerabilityIndicatorPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        Basic style for small indicator which displays a number representing a count of violations, and a short pre-set
+        text string which displays the vulnerability severity.
+      </p>
+      <p className="nx-p">
+        Three basic layouts have been demonstrated below. It is expected that one of these should satisfy any
+        requirment.
+      </p>
+      <table className="nx-table nx-table--gallery-props">
+        <thead>
+          <tr className="nx-table-row">
+            <th className="nx-cell nx-cell--header">Class</th>
+            <th className="nx-cell nx-cell--header">Location</th>
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-vulnerability-indicator-container</code></td>
+            <td className="nx-cell">Container</td>
+            <td className="nx-cell">Basic container class. It's always used with one of the modifiers below.</td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-vulnerability-indicator-container--row</code></td>
```

Comment:
I thought we were going to try to implement it so that it automatically lays out based on the available width.  Granted I know we weren't sure that would be possible (to get 2x2 without allows 3x2 for instance).

Reply:
> I thought we were going to try to implement it so that it automatically lays out based on the available width. Granted I know we weren't sure that would be possible (to get 2x2 without allows 3x2 for instance).

I recall that being mentioned, I didn't recall it being made a requirement. It might be possible with a media query.